zipkin-ui/templates/trace.mustache

```diff
@@ -4,6 +4,9 @@
     <li class=''><a href='#'><strong>Services:</strong> <span class='badge'>{{services}}</span></a></li>
     <li class=''><a href='#'><strong>Depth:</strong> <span class='badge'>{{depth}}</span></a></li>
     <li class=''><a href='#'><strong>Total Spans:</strong> <span class='badge'>{{totalSpans}}</span></a></li>
+    {{#logsUrl}}
```

Comment:
not sure there's an existing class to cover this, so one would need to be made.. I can help think of one, or maybe the notes I've made so far can get you there?